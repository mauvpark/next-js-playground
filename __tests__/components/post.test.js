import renderer from "react-test-renderer";
import Post from "../../components/post";
import postsData from "../../__mocks__/posts.mock";

it("<Post /> should render posts mock datas' index 0", () => {
	const tree = renderer.create(<Post post={postsData()[0]} />).toJSON();
	expect(tree).toMatchSnapshot();
});
