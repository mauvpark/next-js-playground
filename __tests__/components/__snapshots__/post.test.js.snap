// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Post /> should render posts mock datas' index 0 1`] = `
<div>
  <h2
    className="title"
  >
    sunt aut facere repellat provident occaecati excepturi optio reprehenderit
     written by: (
    1
    )
  </h2>
  <p
    className="description"
  >
    quia et suscipit
suscipit recusandae consequuntur expedita et cum
reprehenderit molestiae ut ut quas totam
nostrum rerum est autem sunt rem eveniet architecto
  </p>
</div>
`;
